frameworks/angular/workspace/src/app/services/settings.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from "@angular/core";
+import { type App } from "@openfin/workspace";
+import { BehaviorSubject, from, map, type Observable, tap } from "rxjs";
+import { type CustomSettings, type ManifestWithCustomSettings, type PlatformSettings } from "./types";
+
+@Injectable({ providedIn: "root" })
+export class SettingsService {
+	private readonly apps$ = new BehaviorSubject<App[]>([]);
+
+	/**
+	 * Reads the custom settings from the manifest.fin.json
+	 * @returns The custom settings from the manifest.
+	 */
+	getManifestCustomSettings(): Observable<{
+		platformSettings: PlatformSettings;
+		customSettings?: CustomSettings;
+	}> {
+		const app = fin.Application.getCurrentSync();
+		return from(app.getManifest()).pipe(
+			map((manifest: ManifestWithCustomSettings) => ({
+				platformSettings: {
+					id: manifest.platform?.uuid ?? "",
```

Comment:
Can we provide fin.me.identity.uuid as the default id if not specified? This will prevent multiple instances clashing.